.github/workflows/build_release.yml

```diff
@@ -0,0 +1,54 @@
+name: "Build and push Docker image"
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build_docker_image:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ghcr.io/jorenn92/maintainerr
```

Comment:
```suggestion
            ghcr.io/${{ github.repository }}
```

Reply:
Would that work in this case? Since afaik images need to be lowercased

Reply:
Ah yeah good point. I'm not sure. Probably best to leave it the way it is then.